Bottom: 47f8fbb306d277301161e94470a45895de0fbb36
Top:    b5df9cc04d6e7629ae937bfe9bbc3c7da5193ea0
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-07-26 23:32:18 -0700

Refresh of mpi-finalize

---

diff --git a/src/rsocket.c b/src/rsocket.c
index 3ba0a74..1e391b3 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -1741,9 +1741,7 @@ int rshutdown(int socket, int how)
 				RS_CTRL_SHUTDOWN : RS_CTRL_DISCONNECT;
 		}
 		if (!rs_can_send_ctrl(rs)) {
-			printf("  rshutdown - need ctrl msg %d - %p\n", socket, rs);
 			ret = rs_process_cq(rs, 0, rs_can_send_ctrl);
-			printf("  rshutdown - have ctrl msg %d - %p\n", socket, rs);
 			if (ret)
 				return ret;
 		}
